src/docs/get-started/codelab-web.md

```diff
@@ -301,40 +393,52 @@ and update the app’s UI when the form is complete.
 
 {{site.alert.note}}
   This example does **not** validate the accuracy of the user input.
-  That is something you can add later, if you like.
+  That is something you can add later using form validation, if you like.
 {{site.alert.end}}
 
 <ol markdown="1">
-<li markdown="1">Add a `_formCompleted` field.<br>
-At the top of the `_SignUpFormState` class, add a boolean
-field that you'll set to true when all three fields contain text.
+<li markdown="1">Add an _updateFormProgress method
+In the `_SignUpFormState` class, after the fields are defined add a new
+method called _updateFormProgress():
```

Comment:
Rephrase this sentence, as it doesn't make sense currently with Add "_updateFormProgress twice". Also, codefont `_updateFormProgress`.

Reply:
fixed!